Makefile

```diff
@@ -0,0 +1,39 @@
+#!make
+#----------------------------------------
+# Settings
+#----------------------------------------
+.DEFAULT_GOAL := help
+#--------------------------------------------------
+# Variables
+#--------------------------------------------------
+PYTHON_FILES?=$$(find python -name '*.py')
+DOCKER_COMPOSE_FILE?="docker-compose.yaml"
+#--------------------------------------------------
+# Targets
+#--------------------------------------------------
+bootstrap: ## Installs requirements (python linter & formatter)
+	@pip install flake8 black
+fmt: ## Formats python files
+	@echo "==> Formatting files..."
+	@black $(PYTHON_FILES)
+	@echo ""
+check: ## Checks code for linting/construct errors
+	@echo "==> Checking if files are well formatted..."
+	@flake8 $(PYTHON_FILES)
+	@echo "    [✓]\n"
+build: ## Builds the docker-compose environment
+	@echo "==> Building docker image..."
+	@docker-compose -f ${DOCKER_COMPOSE_FILE} build
+	@echo "    [✓]\n"
+run: build ## Runs & builds the docker environment
+	@docker-compose up
```

Comment:
Missing `-f ${DOCKER_COMPOSE_FILE}`

Reply:
it works for me locally without the -f

Reply:
@cosa65 `docker-compose.yaml` is the default, so I rarely use the `-f` param. But if you had specified a different value of `DOCKER_COMPOSE_FILE` it would be ignored by this command

Reply:
ohh ok, thanks for the explanation!

Reply:
fixed